### DISCUSSION TITLE
Can we change the directory where cp editor compiles code?

### DISCUSSION THREAD

**Discussion by prakharrai1609** [cpeditor/cpeditor#1003] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

It is compiling code in temp folder where my anti virus is hindering it causing it to slow down and hang
How can i change the directory where the files from code companion extension are compiled or placed .

### Expected Behavior

Compiles in the cp editor directory itself

### Steps To Reproduce

Compile

### CP Editor Version

>= 6.8.

### OS

Windows 10

### Are you willing to contribute to this?

- [ ] I'm ready to fix this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [ ] No, thanks

**Comment by ouuan** [comment, marked as answer]
I assume that "the directory where cp editor compiles code" refers to the path of the compilation output, i.e. the executable file.

You can change the path in the [C++ Executable File Path](https://cpeditor.org/docs/preferences/language/#c-executable-file-path) setting.
